src/Annotations/Command/QueueImportCommand.php

```diff
@@ -0,0 +1,144 @@
+<?php
+
+namespace eLife\Annotations\Command;
+
+use eLife\ApiSdk\ApiSdk;
+use eLife\Bus\Limit\Limit;
+use eLife\Bus\Queue\InternalSqsMessage;
+use eLife\Bus\Queue\WatchableQueue;
+use eLife\Logging\Monitoring;
+use Iterator;
+use Psr\Log\LoggerInterface;
+use RuntimeException;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Exception\InvalidArgumentException;
+use Symfony\Component\Console\Helper\ProgressBar;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Throwable;
+
+final class QueueImportCommand extends Command
+{
+    private static $supports = ['profiles'];
+
+    private $sdk;
+    private $serializer;
+    private $output;
+    private $logger;
+    private $monitoring;
+    private $queue;
+    private $limit;
+
+    public function __construct(
+        ApiSdk $sdk,
+        WatchableQueue $queue,
+        LoggerInterface $logger,
+        Monitoring $monitoring,
+        Limit $limit,
+        $supports = []
```

Comment:
Not sure what this is for?

Reply:
This will have more relevance when the command in adopted into `bus-sdk`. For a project such as `annotations` we only want to import profiles into the queue. But if left blank then all the possible content that could be queued will be imported.

Reply:
Only looks to work with profiles though? (`importProfiles`)

Reply:
I agree it should eventually go upstream in `bus-sdk`, no opinion on whether to leave the parameter in now or to drop it to use the constant.

Reply:
I can drop it now and introduce when I add the command to `bus-sdk`.